internal/cli/run.go

```diff
@@ -205,6 +210,21 @@ func runController() {
 	checkf("failed to close ssh connections: %v", err)
 }
 
+func checkHostConfigLocations(location string) error {
```

Comment:
`s/checkHostConfigLocations/checkHost/` (since the first argument's name is `location` and is called with `cfg.Location`)